dev/TEAL.md

```diff
@@ -484,7 +493,8 @@ Some of these have immediate data in the byte or bytes after the opcode.
 | 60 | CreatedAssetID | uint64 | v5  | Asset ID allocated by the creation of an ASA (only with `itxn` in v5). Application mode only |
 | 61 | CreatedApplicationID | uint64 | v5  | ApplicationID allocated by the creation of an application (only with `itxn` in v5). Application mode only |
 | 62 | LastLog | []byte | v6  | The last message emitted. Empty bytes if none were emitted. Application mode only |
-| 63 | StateProofPK | []byte | v6  | 64 byte state proof public key commitment.|
+| 63 | StateProofPK | []byte | v6  | 64 byte state proof public key commitment |
```

Comment:
Is it for key registration transaction
```suggestion
| 63 | StateProofPK | []byte | v6  | 64 byte state proof public key |
```
I'm afraid that the word "commitment" can be confusing. The fact that it is a commitment is mostly not relevant to the developer.